### PR TITLE
Option to add button and additional text to header

### PR DESCRIPTION
**Pull request by sdevalapurkar-bcgov**
# Description

Option to add button and additional text to header

## Screenshots

header-v2

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Local and storybook

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -43,8 +45,16 @@ export const Header = ({ header: { name, history } }) => {
...
-          {name}
+        <div className="bcgov-flex-spacing">
+          <div id="title" className="navbar-brand bcgov-vertical-center">
+            {name}
+          </div>
+          <div>
+            {additionalText && (
+              <div className="navbar-brand">{additionalText}</div>
+            )}
+            {button && <div className="navbar-brand">{button()}</div>}
```
Comment (alexjoybc) [inline src/components/header/Header.js:56]: nice!!

**Review by alexjoybc — COMMENTED** [review]
you should mention fix issue #250

**Review by alexjoybc — APPROVED** [review]
Awesome. looks good!

File: src/components/header/Header.js (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply src/components/header/Header.js:56]: thanks!

**Pull request merged**